src/stdatamodels/jwst/datamodels/pictureframe.py

```diff
@@ -0,0 +1,38 @@
+from stdatamodels.dynamicdq import dynamic_mask
+from .dqflags import pixel
+from .reference import ReferenceFileModel
+
+
+__all__ = ["PictureFrameModel"]
+
+
+class PictureFrameModel(ReferenceFileModel):
+    """
+    A data model for 2D thermal picture frame reference files.
+
+    Parameters
+    __________
+    data : numpy float32 array
+         The science data
+
+    dq : numpy uint32 array
+         Data quality array
+
+    err : numpy float32 array
+         Error array
+
+    dq_def : numpy table
+         DQ flag definitions
+
+    """
+
+    schema_url = "http://stsci.edu/schemas/jwst_datamodel/pictureframe.schema"
+
+    def __init__(self, init=None, **kwargs):
+        super(PictureFrameModel, self).__init__(init=init, **kwargs)
+
+        self.dq = dynamic_mask(self, pixel)
+
+        # Implicitly create arrays
+        self.dq = self.dq
+        self.err = self.err
```

Comment:
Are we sure that having a zero-filled error array is what we want in all cases where the model is initialized without an error array? That's what this line will enforce.  If the line is removed, the error array will just not be populated until assigned.